cmd/commands/restore.go

```diff
@@ -23,9 +23,10 @@ import (
 
 // RestoreCmd represents the restore commands
 var RestoreCmd = &cobra.Command{
-	Use:   "restore-deleted",
-	Short: "Restores a CR that was accidentally deleted and is still in terminating state. Ex: restore cephcluster <my-cluster>",
-	Args:  cobra.MinimumNArgs(1),
+	Use:     "restore-deleted",
+	Short:   "Restores a CR that was accidentally deleted and is still in terminating state. Ex: restore cephcluster <my-cluster>",
```

Comment:
```suggestion
	Short:   "Restores a CR that was accidentally deleted and is still in terminating state.,
```

remove this as not needed anymore

Reply:
updated.